sqlup-mode.el

```diff
@@ -188,7 +187,7 @@ the given DIALECT of SQL."
   "Depending on the major mode (redis-mode or sql-mode), find the
 correct keywords. If not, create a (hopefully sane) default based on
 ANSI SQL keywords."
-  (cond ((sqlup-redis-mode-p) (mapcar 'downcase (sqlup-get-redis-keywords)))
+  (cond ((derived-mode-p 'redis-mode) (mapcar 'downcase (sqlup-get-redis-keywords)))
```

Comment:
Just because it's a fairly simple and idiomatic function call that is only used in one place, so I didn't seeany need to wrap it in another function. I don't have a strong opinion about it though.